codebase/superdarn/src.bin/tk/tool/make_grid.2.0/make_grid.c

```diff
@@ -514,6 +517,9 @@ int main(int argc,char *argv[]) {
         if (tolower(chnstr_fix[0])=='d') channel_fix=4;
     }
 
+    if (catflg == 0) 
+        fprintf(stderr, "Deprecation Warning: -c option will removed next release. This will not change the functionality of make_grid and will be able to read multiple fitacf files of one radar still\n");
```

Comment:
```suggestion
        fprintf(stderr, "Deprecation Warning: -c option will be removed in the next release. This will not change the functionality of make_grid. Multiple input files from one radar are now detected and concatenated automatically.\n");
```